### PR TITLE
Add pingsutw to OWNERS

### PR DESCRIPTION
**Pull request by pingsutw**
I’d like to help review the code

### CODE REVIEW

**Comment by jlewi** [comment]
@pingsutw Can you start by listing yourself as a reviewer and then as you get more familiar with the code we can make you an approver?

**Comment by pingsutw** [comment]
@jlewi sure, It makes sense.
It's my pleasure to help to review the PR.

File: OWNERS [hunk 1]
```diff
@@ -1,4 +1,5 @@
 approvers:
 - bobgy
 - jlewi
+- pingsutw
```
Comment (jlewi) [inline OWNERS:4]: Move to reviewers 
```suggestion
- pingsutw
```

File: OWNERS [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 - rmgogogo
```
Comment (jlewi) [inline OWNERS:5]:
```suggestion
- rmgogogo
reviewers:
- pingsutw
```

**Comment by Bobgy** [comment]
/approve
the label was removed... github doesn't handle concurrent label management well

**Pull request merged**